wheel/src/api.rs

```diff
@@ -1,5 +1,5 @@
 use crate::compression;
-use crate::run_generator::{PySpend, PySpendBundleConditions, __pyo3_get_function_run_generator};
+use crate::run_generator::{PySpend, PySpendBundleConditions, __pyo3_get_function_run_generator, __pyo3_get_function_run_block_generator};
```

Comment:
These imports are ugly. I take the blame for it. Ultimately (not now) we could create a function in `run_generator.rs` called `add_to_api` which takes a module and does the `m.add_function(wrap_pyfunction!(run_block_generator, m)?)?;` stuff there, so we don't have to resort to these name-mangled imports.

Reply:
We could also consider submodules. See https://github.com/Chia-Network/chia_rs/blob/main/wheel/src/compression.rs#L13 for an example (`import chia_rs.compression`).